src/test/java/TestVector.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestVector {
+
+    Vector testVector1 = new Vector(1, 2, 3);
+    Vector testVector2 = new Vector(2, 2, 2);
+    Vector testVector3 = new Vector(7, 8, 9);
+
+
+    @Test
+    public void testVector() {
+
+        // check that constructor works well
+        assertEquals(8, testVector3.getCoordY());
```

Comment:
let's start add error message for all asserts as the last argument

Reply:
done